src/main/kotlin/com/hibob/academy/dao/PetDao.kt

```diff
@@ -63,4 +64,16 @@ class PetDao(private val sql: DSLContext) {
             .where(petsTable.id.eq(petId))
             .execute()
     }
+
+    fun countPetsByType(companyId: Long): Map<String, Int> {
```

Comment:
what about the second func of Get pets by owner: Given an owner ID, return all the information of the pets this owner adopted.